docs/tutorials/google/calibration_api.ipynb

```diff
@@ -134,61 +134,9 @@
     "project_id = '' #@param {type:\"string\"}\n",
-    "\n",
-    "\n",
-    "use_noisy_simulator = False\n",
-    "if project_id == '' and 'GOOGLE_CLOUD_PROJECT' not in os.environ:\n",
-    "    print(\"No project_id provided and environment variable \"\n",
-    "          \"GOOGLE_CLOUD_PROJECT not set.\")\n",
-    "    use_noisy_simulator = True\n",
-    "else: \n",
-    "    os.environ['GOOGLE_CLOUD_PROJECT'] = project_id\n",
-    "\n",
-    "    def authenticate_user():\n",
-    "        \"\"\"Runs the user through the Colab OAuth process.\n",
-    "\n",
-    "        Checks for Google Application Default Credentials and runs interactive login \n",
-    "        if the notebook is executed in Colab. In case the notebook is executed in Jupyter notebook\n",
-    "        or other IPython runtimes, no interactive login is provided, it is assumed that the \n",
-    "        `GOOGLE_APPLICATION_CREDENTIALS` env var is set or `gcloud auth application-default login`\n",
-    "        was executed already.\n",
-    "\n",
-    "        For more information on using Application Default Credentials see \n",
-    "        https://cloud.google.com/docs/authentication/production\n",
-    "        \"\"\"\n",
-    "        in_colab = False\n",
-    "        try:\n",
-    "            from IPython import get_ipython\n",
-    "            in_colab = 'google.colab' in str(get_ipython())\n",
-    "        except: \n",
-    "            # Notebook is not executed within IPython. Assuming external authentication.\n",
-    "            return \n",
-    "\n",
-    "        if in_colab: \n",
-    "            from google.colab import auth      \n",
-    "            print(\"Getting OAuth2 credentials.\")\n",
-    "            print(\"Press enter after entering the verification code.\")\n",
-    "            auth.authenticate_user(clear_output=False)\n",
-    "            print(\"Authentication complete.\")\n",
-    "        else: \n",
-    "            print(\"Notebook is not executed with Colab, assuming Application Default Credentials are setup.\") \n",
-    "\n",
-    "    authenticate_user()\n",
-    "    print(\"Successful authentication to Google Cloud.\")\n",
-    "\n",
     "processor_id = \"\" #@param {type:\"string\"}\n",
-    "if use_noisy_simulator or processor_id == \"\":\n",
-    "    print(\"Using a noisy simulator.\")\n",
-    "    sampler = cg.PhasedFSimEngineSimulator.create_with_random_gaussian_sqrt_iswap(\n",
-    "        mean=cg.SQRT_ISWAP_INV_PARAMETERS,\n",
-    "        sigma=cg.PhasedFSimCharacterization(\n",
-    "            theta=0.01, zeta=0.10, chi=0.01, gamma=0.10, phi=0.02\n",
-    "        ),\n",
-    "    )\n",
-    "    device = cg.Bristlecone\n",
-    "else:\n",
-    "    device = cg.get_engine_device(processor_id)\n",
-    "    sampler = cg.get_engine_sampler(processor_id, gate_set_name=\"sqrt_iswap\")"
+    "\n",
+    "(device, line_length), sampler, _ = cg.engine.engine_sampler.get_device_sampler(project_id, processor_id)"
```

Comment:
line_length is unused. I see you use `_` for the final unused variable, so you can likely use it here.

But also: see comment above. I don't think we should have line length as part of this function

Reply:
I removed line length and only left it in the Floquet notebook now.